### PR TITLE
Fixes #58 : Make movie tiles round

### LINKED ISSUE #58: Make movie tiles round
Make the size of the movie tiles smaller and shape it round

### PR DESCRIPTION
**Pull request by thenullterminator**
* Please fill out the template below

### Did you read the instructions in [README](../README.md) ?

>Yes
### Solves which issue ?

>Solves #[issue number]
### 58

>[Paste the link here]
### https://thenullterminator.github.io/Selena-Gomez-Discography22/

>[Write Here. Be Specific]

### CODE REVIEW

File: css/theme.css [hunk 1]
```diff
@@ -3,22 +3,34 @@ body {
   padding-bottom: 30px;
 }
 
+/*Readjusting for mobile view"*/
```
Comment (OrionStar25) [inline css/theme.css]: There's no need for this class. Its not solving any problem.

File: css/theme.css [hunk 2]
```diff
@@ -65,6 +77,13 @@ body {
   width: auto;
 }
 
+/*To make the movie tiles round*/
```
Comment (OrionStar25) [inline css/theme.css:65]: This class is correct.

File: index.html [hunk 3]
```diff
@@ -77,18 +78,25 @@ <h1> <b>|| Selena Marie Gomez ||</b></h1>
 
      <div class="row row-offcanvas row-offcanvas-right">
 
-        <div class="col-xs-12 col-sm-9 " class="float-none1">
+        <div class="col-xs-12 col-sm-9 " id="float-none1">
```
Comment (OrionStar25) [inline index.html]: This needs to be a class. Please sync your fork and then make your changes on top of it. You would need to `rebase`

File: index.html [hunk 4]
```diff
@@ -77,18 +78,25 @@ <h1> <b>|| Selena Marie Gomez ||</b></h1>
...
-              <p> <img src="img/revival.jpg"> </p>
-      			 <div class="dropdown">
-  				<button class="dropbtn">Listen</button>
- 				 <div class="dropdown-content">
+				
+
+
+            <div class="album col-xs-6 col-lg-4" id="float-none" >
+
+				<h2><i class="readjust">Revival</i></h2>
```
Comment (OrionStar25) [inline index.html]: Please remove all this. Not necessary in this PR.

File: index.html [hunk 5]
```diff
@@ -219,12 +291,108 @@ <h2><i>Kiss And Tell</i></h2>
...
+			                <br>
+					<br>
+					<h1 style = "font-size: 45px;text-align:center;" > Movies </h1>
+					
+					
+					
+						<div class="album col-xs-6 col-lg-4">
+						<h3>Neighbours 2</h3>
+						
+						<p><img src="img/neighbors2.jpeg " class="roundmovietiles" ></p>
```
Comment (OrionStar25) [inline index.html]: Your fork isn't upto date with mine. What you're doing is adding another set of movie tiles to the webpage. 
Update your fork. Then add your `roundtiles` class to that. That will work. (Rn it shows you've added movies on top of movies)

**Review by OrionStar25 — CHANGES_REQUESTED** [review]

**Comment by thenullterminator** [comment]
Changes Done.(Sorry for the inconvenience)

**Review by OrionStar25 — APPROVED** [review]
No worries! Good work. Everyone learns through mistakes :smile:  Lets hope it works this time

**Pull request merged**